paimon-web-ui/src/store/config/type.ts

```diff
@@ -16,5 +16,5 @@ specific language governing permissions and limitations
 under the License. */
 
 export type Theme = 'dark' | 'light'
-export type NavBar = 'playground' | 'cdc' | 'system' | 'settings'
+export type NavBar = 'playground' | 'cdc' | 'system' | 'settings' | 'null'
```

Comment:
Why does `NavBar` have null value?

Reply:
The front-end menu now incorporates permission management, which may result in users not having any selectable menus based on their permissions.

Reply:
@hdygxsj, I don’t think this is the reason why introduce null bar.

Reply:
Made some minor adjustments.